File: LATIHAN1DPBO2023/CPP/Mhs.cpp
//import library
#include <iostream>
#include <string>

// menggunakan standard namespace
using namespace std;

// deklarasi class human
class Mhs
{
    // atribut privat
    private:
        string name;
        string nim;
        string ps;
        string fakultas;
    
    // metode publik
    public:
        /*konstruktor*/
        Mhs() //-> konstruktor tanpa parameter
        {
            // set variabel name dan gender ke default
            this->name = "";
            this->nim = "";
            this->ps = "";
            this->fakultas = "";
        }
        /*Getter dan Setter*/

        // get name
        string get_name()
        {
            return this->name;
        }
        // set name
        void set_name(string name)
        {
            this->name = name;
        }
        // get nim
        string get_nim()
        {
            return this->nim;
        }
        // set gender
        void set_nim(string nim)
        {
            this->nim = nim;
        }
        // get program_studi
        string get_ps()
        {
            return this->ps;
        }
        // set program_studi
        void set_ps(string ps)
        {
            this->ps = ps;
        }
        // get fakultas
        string get_fakultas()
        {
            return this->fakultas;
        }
        // set fakultas
        void set_fakultas(string fakultas)
        {
            this->fakultas = fakultas;
        }
        /*end getter dan setter*/
        
        /*desktruktor*/
        ~Mhs() //-> default destruktor
        {

        }
};

File: LATIHAN1DPBO2023/PYTHON/Mhs.py
# class declaration
class Mhs:
    # __ private
    # _ public
    __name = ""
    __nim =""
    __ps = ""
    __fakultas = ""
    
    def __init__(self, name = "", nim = "", ps = "", fakultas = ""):
        self.__name = name
        self.__nim = nim
        self.__ps = ps
        self.__fakultas = fakultas
    # getter dan setter #
    
    # get name
    def get_name(self):
        return self.__name
    # set name
    def set_name(self, name):
        self.__name = name
    # get nim
    def get_nim(self):
        return self.__nim
    # set nim
    def set_nim(self, nim):
        self.__nim = nim
    # get ps
    def get_ps(self):
        return self.__ps
    # set ps
    def set_ps(self, ps):
        self.__ps = ps
    # get fakultas
    def get_fakultas(self):
        return self.__fakultas
    # set fakultas
    def set_fakultas(self, fakultas):
        self.__fakultas = fakultas